pkgs/compiler-lib/compiler/commands/test.rkt

```diff
@@ -1092,8 +1093,12 @@
  [("--table" "-t")
   "Print a summary table"
   (set! table? #t)]
+ #:once-each
+ [("--output" "-o") filename
+  "Save stdout and stderr to file"
+  (set! default-output-file filename)]
  #:args file-or-directory-or-collects-or-pkgs
- (let ()
+ (define (test-main)
```

Comment:
What's the reason behind calling this `test-main`, it's quite surprising to me as it is not a test.

Reply:
Any suggestions for the name? (main)?

My thoughts, the test.rkt provides the main function of test, specificly, `raco test`, that's why I calling it as `test-main`